src/objects.rs

```diff
@@ -1848,51 +1849,28 @@ mod tests {
         assert!(de("00:00:AA").is_err());
     }
 
-    fn nearly_equal(x: f64, y: f64, epsilon: f64) -> bool {
-        if x == y {
-            true
-        } else {
-            let normalized_delta = (x - y).abs() / y;
-            normalized_delta < epsilon
-        }
-    }
-
-    const TOLERANCE: f64 = 0.001;
-
     // distance between COORD1 and COORD2 is 357.64 from
     // https://gps-coordinates.org/distance-between-coordinates.php
     const COORD1: Coord = Coord {
-        lon: 2.377054,
-        lat: 48.846995,
+        lon: 2.377_054,
+        lat: 48.846_995,
     };
     const COORD2: Coord = Coord {
-        lon: 2.374377,
-        lat: 48.844304,
+        lon: 2.374_377,
+        lat: 48.844_304,
     };
 
     #[test]
     fn orthodromic_distance() {
-        assert!(nearly_equal(COORD1.distance_to(&COORD1), 0.0, TOLERANCE));
-        assert!(nearly_equal(COORD1.distance_to(&COORD2), 357.64, TOLERANCE));
-        assert!(nearly_equal(COORD2.distance_to(&COORD1), 357.64, TOLERANCE));
+        relative_eq!(COORD1.distance_to(&COORD1), 0.0);
```

Comment:
I believe it should be `assert_relative_eq!` because `relative_eq!` only returns a `bool` (which will never fail).

Reply:
done

Reply:
You might have missed one, see https://github.com/CanalTP/transit_model/pull/604#discussion_r409545688.